app/js/main.js

```diff
@@ -131,7 +131,12 @@ document.addEventListener('DOMContentLoaded', function () {
           console.log(coords);
           locationProvider.updateLocation(coords.latitude, coords.longitude);
           locationProvider.updateOrientation({ alpha: coords.heading });
-
+          
+          
+          (async () => {
+            var road = await getCurrentRoad(locationProvider);
+            console.log(road)
+          })();
```

Comment:
I was going to suggest adding this to the audio callout queue rather than just dumping it to the console, but that's not trivial to do here with the current code organization.

Reply:
On second thought, does something like this work?
```suggestion
          // Speak nearest road
          (async () => {
            var road = await getCurrentRoad(locationProvider);
            audioQueue.addToQueue({ soundUrl: 'app/sounds/sense_mobility.wav' })
            audioQueue.addToQueue({ text: `Nearest road: ${road[0].properties.name}` })
          })();
```